src/lib/bit/bit.h

```diff
@@ -53,6 +53,7 @@ extern "C" {
 /** @cond false **/
 #define bit_likely(x)    __builtin_expect((x),1)
 #define bit_unlikely(x)  __builtin_expect((x),0)
+
```

Comment:
Unrelated change in this commit. It's removed in the next one but better keep each commit clean.

Also, the commit message looks mangled:

```
Required by Tarantool-EE PR #1297Required by Tarantool-EE PR [https://github.com/tarantool/tarantool/issues/1285](https://github.com/tarantool/tarantool-ee/pull/1285)
```